engine/src/main/java/org/terasology/engine/module/ExternalApiWhitelist.java

```diff
@@ -42,6 +44,7 @@
             .add("java.awt")
             .add("java.awt.geom")
             .add("java.awt.image")
+            .add("javafx.util")
```

Comment:
Should be removed as well if JavaFX is not required.

Reply:
JavaFX was removed from more recent Java versions, so would want to consider either avoid it or bringing in the newer standalone javafx - avoiding would be better.